core/src/main/java/org/eclipse/hono/config/ServerConfig.java

```diff
@@ -275,4 +277,36 @@ public final void setSni(final boolean sni) {
     public final boolean isSni() {
         return this.sni;
     }
+
+    /**
+     * Gets the idle timeout.
+     * <p>
+     * A connection will timeout and be closed if no data is received or sent within the idle timeout period.
+     * A zero value means no timeout is used.
+     * <p>
+     * The default value is {@code 60} in seconds.
+     *
+     * @return The idle timeout in seconds.
+     */
+    public final int getIdleTimeout() {
```

Comment:
Looks like this class and the *AmqpAdapterProperties* classes have a different understanding of the semantics of this property. First, this class returns #seconds while *AmqpAdapterProperties.getIdleTimeout()* seems to return #millis. Second, in *AmqpAdapterProperties* the *idleTimeout* is used to keep an AMQP connection to a device alive whereas in this class, the property is supposed to be used for terminating a device's HTTP request.
FMPOV we need to keep the *AmqAdapterProperties* unchanged and instead should move the *idleTimeout* property from this class to both *HttpProtocolAdapterProperties* and *HttpServiceConfigProperties* ...

@calohmn WDYT?

Reply:
Oh, I missed that. 
> FMPOV we need to keep the *AmqAdapterProperties* unchanged and instead should move the *idleTimeout* property from this class to both *HttpProtocolAdapterProperties* and *HttpServiceConfigProperties* ...

I agree.

Reply:
Ok. I'll change it :)